### PR TITLE
Fixes 216: Restructure frontend components

### PR DESCRIPTION
**Pull request by Andrewgdewar**
This cannot be merged until the following are merged: 

[Fixes 208: Add api validation for gpg-keys](https://github.com/content-services/content-sources-frontend/pull/38)

[Fixes 76: Allow Pasting of URL for GPGKey using API](https://github.com/content-services/content-sources-frontend/pull/35)  

** Additional Changes: Fixed a bug related to pasting a GPG_key url and the radio below not being automatically set.

### CODE REVIEW

**Comment by jlsherrill** [comment]
https://issues.redhat.com/browse/HMSCONTENT-216

**Review by rverdile — COMMENTED** [review]
this seems okay to me. I'll come back and double check when the remaining dependency is merged

File: src/components/ContentListTable/components/AddContent/AddContent.tsx [hunk 1]
```diff
@@ -575,7 +575,17 @@ const AddContent = ({ isLoading }: Props) => {
...
+                                  gpgKey: gpgData,
+                                  gpgLoading: false,
+                                  ...(gpgKey === '' && !!value
+                                    ? {
+                                        metadataVerification:
+                                          !!validationList?.[index]?.url
+                                            ?.metadata_signature_present,
+                                      }
+                                    : {}),
+                                });
```
Comment (rverdile) [inline src/components/ContentListTable/components/AddContent/AddContent.tsx:588]: is this change a remnant of some other PR? it doesn't strike me as a "restructure".

Comment (Andrewgdewar) [reply src/components/ContentListTable/components/AddContent/AddContent.tsx:588]: hmm good catch. Ugh.. this needs to be removed.

Comment (Andrewgdewar) [reply src/components/ContentListTable/components/AddContent/AddContent.tsx:588]: Actually. going to check. I'm concerned that the onTextChange below may not work without this, and that's why I added it here. Testing now.

**Review by rverdile — COMMENTED** [review]
It doesn't seem like gpg key validation is working here
image

**Comment by rverdile** [comment]
To produce that I clicked "Add 20", then typed gibberish into the gpg key field for one of them.

**Review by rverdile — APPROVED** [review]
i was on the wrong backend PR, this lgtm

**Pull request merged**